benchopt/tests/__init__.py

```diff
@@ -11,6 +11,7 @@
 # Pattern to select specific datasets or solvers.
 SELECT_ONE_SIMULATED = r'simulated*500*rho=0\]'
 SELECT_ONE_PGD = r'python-pgd*step_size=1\]'
+SELECT_ONE_OBJECTIVE = r'dummy sparse regression*reg=0.1\]'
```

Comment:
To avoid issues with spaces in the test:
```suggestion
SELECT_ONE_OBJECTIVE = r'dummy*reg=0.1\]'
```